core/src/main/java/com/nerds/gamejam/ecs/system/PlanetViewGUISystem.java

```diff
@@ -2,29 +2,54 @@
 
 import com.artemis.BaseSystem;
 import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.Texture;
```

Comment:
All this code should move to the bootstrap system as you are in effect bootstrapping the monster, this class is for gui stuff ;-)